lib/plugins/aws/metrics/awsMetrics.js

```diff
@@ -55,160 +52,189 @@ class AwsMetrics {
+    });
   }
 
   showMetrics(metrics) {
     let message = '';
 
-    message += `${chalk.yellow.underline(this.options.function)}\n`;
+    if (this.options.function) {
+      message += `${chalk.yellow.underline(this.options.function)}\n`;
+    } else {
+      message += `${chalk.yellow.underline('Service wide metrics')}\n`;
+    }
 
     const formattedStartTime = moment(this.options.startTime).format('LLL');
     const formattedEndTime = moment(this.options.endTime).format('LLL');
     message += `${formattedStartTime} - ${formattedEndTime}\n\n`;
 
     if (metrics && metrics.length > 0) {
+      let invocations = 0;
+      let throttles = 0;
+      let errors = 0;
+      let duration = 0;
+
       _.forEach(metrics, (metric) => {
-        if (metric.Label === 'Invocations') {
-          const datapoints = metric.Datapoints;
-          const invocations = datapoints
-            .reduce((previous, datapoint) => previous + datapoint.Sum, 0);
-          message += `${chalk.yellow('Invocations:', invocations, '\n')}`;
-        } else if (metric.Label === 'Throttles') {
-          const datapoints = metric.Datapoints;
-          const throttles = datapoints
-            .reduce((previous, datapoint) => previous + datapoint.Sum, 0);
-          message += `${chalk.yellow('Throttles:', throttles, '\n')}`;
-        } else if (metric.Label === 'Errors') {
-          const datapoints = metric.Datapoints;
-          const errors = datapoints
-            .reduce((previous, datapoint) => previous + datapoint.Sum, 0);
-          message += `${chalk.yellow('Errors:', errors, '\n')}`;
-        } else {
-          const datapoints = metric.Datapoints;
-          const duration = datapoints
-            .reduce((previous, datapoint) => previous + datapoint.Average, 0);
-          const formattedRoundedAvgDuration = `${Math.round(duration * 100) / 100}ms`;
-          message += `${chalk.yellow('Duration (avg.):', formattedRoundedAvgDuration)}`;
-        }
+        _.forEach(metric, (funcMetric) => {
+          if (funcMetric.Label === 'Invocations') {
+            const datapoints = funcMetric.Datapoints;
+            invocations += datapoints
+              .reduce((previous, datapoint) => previous + datapoint.Sum, 0);
+          } else if (funcMetric.Label === 'Throttles') {
+            const datapoints = funcMetric.Datapoints;
+            throttles += datapoints
+              .reduce((previous, datapoint) => previous + datapoint.Sum, 0);
+          } else if (funcMetric.Label === 'Errors') {
+            const datapoints = funcMetric.Datapoints;
+            errors += datapoints
+              .reduce((previous, datapoint) => previous + datapoint.Sum, 0);
+          } else {
+            const datapoints = funcMetric.Datapoints;
+            duration += datapoints
+              .reduce((previous, datapoint) => previous + datapoint.Average, 0);
+          }
+        });
       });
+      // format the duration
```

Comment:
uncle bob said once: it's always better to create function that add comment :)

`function formatDuration(duration)` would be great here.